.circleci/verbatim-sources/binary-job-specs.yml

```diff
@@ -0,0 +1,242 @@
+  binary_linux_build:
+    <<: *binary_linux_build_params
```

Comment:
NB: There isn't really any need to macro here, since you only use it once, right?

Reply:
Good point! I was just mindlessly copying the structure from the other builds haha.

Reply:
Actually, I think I'm going to leave it in, just so that the structure is regular (even if it's unnecessary)